### PR TITLE
Altered code to use the new utils functions

### PR DESCRIPTION
**Pull request by evalott100**
Closes https://github.com/PandABlocks/PandABlocks-ioc/issues/16

Should only be merged after https://github.com/PandABlocks/PandABlocks-client/pull/52

### CODE REVIEW

**Comment by evalott100** [comment]
Tests failing because pandablocks_client requires release with [this PR](https://github.com/PandABlocks/PandABlocks-client/pull/52)

**Review by AlexanderWells-diamond — APPROVED** [review]
I'm happy with these changes, but I think it would be best to wait for a PandA-client release and update our dependencies before merging this.

**Pull request merged**
